exercises/practice/promises/.meta/config.json

```
@@ -1,5 +1,5 @@
 {
-  "blurb": "Compute the prime factors of a given natural number.",
+  "blurb": "Learn about promises by implementing some common promise functions.",
```

Practice exercises don't really teach anything so I would recommend to rephrase this:
```suggestion
  "blurb": "Practice promises by implementing some common promise functions.",
```